src/OpenTelemetry.Exporter.OpenTelemetryProtocol/OtlpLogExporterHelperExtensions.cs

```diff
@@ -44,29 +43,55 @@ public static OpenTelemetryLoggerOptions AddOtlpExporter(this OpenTelemetryLogge
         Action<OtlpExporterOptions> configure)
         => AddOtlpExporterInternal(loggerOptions, configure);
 
+    /// <summary>
+    /// Adds OTLP Exporter as a configuration to the OpenTelemetry ILoggingBuilder.
+    /// </summary>
+    /// <param name="loggerOptions"><see cref="OpenTelemetryLoggerOptions"/> options to use.</param>
+    /// <param name="configureExporterAndProcessor">Callback action for configuring <see cref="OtlpExporterOptions"/> and <see cref="LogRecordExportProcessorOptions"/>.</param>
+    /// <returns>The instance of <see cref="OpenTelemetryLoggerOptions"/> to chain the calls.</returns>
+    public static OpenTelemetryLoggerOptions AddOtlpExporter(
+        this OpenTelemetryLoggerOptions loggerOptions,
+        Action<OtlpExporterOptions, LogRecordExportProcessorOptions> configureExporterAndProcessor)
```

Comment:
I guess since configuration, options, DI cannot be supported from OpenTelemetryLoggerOptions, thoughts on whether we should communicate this in remarks or something?

@CodeBlanch what do you think? Got a good way to communicate this?

Reply:
I agree this would be good to callout. But maybe on `ILoggingBuilder.AddOpenTelemetry` instead?

```diff
public static class OpenTelemetryLoggingExtensions
{
    /// <summary>
    /// Adds an OpenTelemetry logger named 'OpenTelemetry' to the <see cref="ILoggerFactory"/>.
    /// </summary>
    /// <remarks>
-     /// Note: This is safe to be called multiple times and by library
-     /// authors. Only a single <see cref="OpenTelemetryLoggerProvider"/>
-     /// will be created for a given <see cref="IServiceCollection"/>.
+     /// Notes:
+     /// <list type="bullet">
+     /// <item>This is safe to be called multiple times and by library authors.
+     /// Only a single <see cref="OpenTelemetryLoggerProvider"/> will be created
+     /// for a given <see cref="IServiceCollection"/>.</item>
+     /// <item><see cref="IServiceCollection"/> / <see cref="IServiceProvider"/>
+     /// features (DI, Options, IConfiguration, etc.) are not available when
+     /// using <see cref="ILoggingBuilder"/>.</item>
+     /// </list>
    /// </remarks>
    /// <param name="builder">The <see cref="ILoggingBuilder"/> to use.</param>
    /// <returns>The supplied <see cref="ILoggingBuilder"/> for call chaining.</returns>
    public static ILoggingBuilder AddOpenTelemetry(...) {}
```

Reply:
updated